### PR TITLE
Integrate Lodestar Light Client

### PR DESCRIPTION
**Pull request by acolytec3**
Adds a Lodestar Light Client to the Beacon protocol and implements a new transport layer for it that uses the Portal Network for sync

### CODE REVIEW

**Review by ScottyPoi — APPROVED** [review]
:+1:

**Comment by ScottyPoi** [comment]
Very cool!  
I'm eager to build on top of this, and have a hybrid portal client of some kind to show off
Could you add some documentation?

File: packages/portalnetwork/src/subprotocols/beacon/types.ts [hunk 1]
```diff
@@ -5,6 +5,8 @@ import { Bytes32Type } from '../types.js'
 export const MainnetGenesisValidatorsRoot =
   '0x4b363db94e286120d76eb905340fdd4e54bfe9f06bf33ff6cf5ad27f511bfe95'
 
+export const MainnetGenesisTime = 1606824023
```
Comment (g11tech) [inline packages/portalnetwork/src/subprotocols/beacon/types.ts:8]: you can import this data from the lodestar :
```
 import {networksChainConfig} from "@lodestar/config/networks";
 ```
 networksChainConfig has genesis root and time for all major networks (and gets updated as well)

**Review by g11tech — APPROVED** [review]
Without going deep into ultralight stuff, this looks good to me w.r.t. lightclient integration 🎉

**Pull request merged**